src/pkgcore/config/domain.py

```diff
@@ -40,13 +40,18 @@ class domain:
     def triggers(self):
         return tuple(self._triggers)
 
-    def pkg_operations(self, pkg, observer=None):
+    def get_pkg_operations(self, pkg, observer=None):
```

Comment:
this change is the sole one debatable (imo); I can drop it if strong arguments exist, but per my commit message, these operations should be verbose/declarative as to their intent.

Reply:
As it makes the usage more clear, I agree :)